src/tokens/lst_tokens/token_new.c

```diff
@@ -0,0 +1,28 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   token_new.c                                        :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: mcourtoi <mcourtoi@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/03/08 19:37:40 by mcourtoi          #+#    #+#             */
+/*   Updated: 2023/03/09 15:15:52 by mcourtoi         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "minishell.h"
+
```

Comment:
Please, add a Doxygen comment to briefly explain what this function does and how to use it :slightly_smiling_face: